.github/workflows/freezeyt_blog.yaml

```diff
@@ -16,12 +16,20 @@ jobs:
     name: build blog content
     # Set the type of machine to run on
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Je nutné to spouštět na čtyřech verzích Pythonu?

Reply:
Tohle by mělo otestovat blog samotný, kde si můžeme vybrat jednu verzi a říct že ostatní nepodporujeme.
Jakou používáte verzi Pythonu? Já 3.8, ale brzo přejdu na 3.9.

Reply:
Já také 3.8

Reply:
Já také používám Python 3.8